backend/app/services/doctransform/job.py

```
@@ -232,44 +275,33 @@ def execute_job(
         if callback_url:
             send_callback(callback_url, success_payload, webhook_secret=webhook_secret)
 
+    except (Timeout, SoftTimeLimitExceeded) as err:
+        timeout_err = TimeoutError(f"Task exceeded soft time limit")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)